docs/Extensions.md

```diff
@@ -174,3 +174,15 @@ services:
 x-podman:
     in_pod: false
 ```
+
+It is also possible to override the default arguments for pod creation that are
+used when --pod-args is not passed on the command line:
```

Comment:
Makes sense to document the default too.

Reply:
Done